src/sastadev/conf.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from sastadev import SD_DIR
+from sastadev import sentence_parser
+
+
+# @dataclass
+# class SastadevConfig:
+#     '''Class for keeping track of application configuration'''
+
+#     ALPINO_HOST: str = 'localhost'
+#     ALPINO_PORT: int = 7001
+#     logger = logging
+#     sd_dir: str = SD_DIR
+#     dataroot: str = '.'
+#     parse_func = sentence_parser.parse
+
+#     def __post_init__(self):
+#         print('initted')
+
+
+class SastadevConfig:
+    '''Class for keeping track of application configuration'''
+
+    def __init__(self,
+                 ALPINO_HOST: str = 'localhost',
+                 ALPINO_PORT: int = 7001,
+                 LOGGER=logging,
```

Comment:
Might be better to use `logging.getLogger('sastadev')` instead of the root logger, giving more control to downstream  users

Reply:
Agree. I'll alter the main script a little bit to also work with that configuration.